kong/db/dao/init.lua

```diff
@@ -891,10 +891,12 @@ local function generate_foreign_key_methods(schema)
           return nil, tostring(err_t), err_t
         end
 
-        local _
-        _, err_t = self.strategy:delete_by_field(name, unique_value, options)
+        local rows_affected
+        rows_affected, err_t = self.strategy:delete_by_field(name, unique_value, options)
         if err_t then
           return nil, tostring(err_t), err_t
+        elseif not rows_affected then
```

Comment:
style issue: should add a blank line before `elseif`.

Reply:
This has been fixed. Thanks!